pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -267,7 +297,7 @@ func (r *reconciler) Reconcile(rule *CompletedRule) error {
 	value, exists := r.lastRealizeds.Load(rule.ID)
 	ruleTable := r.getOFRuleTable(rule)
 	priorityAssigner, _ := r.priorityAssigners[ruleTable]
-	if rule.isAntreaNetworkPolicyRule() {
+	if rule.isAntreaNetworkPolicyRule() && !rule.isIGMPEgressPolicyRule() {
```

Comment:
please add comment to explain
"IGMP Egress policy is enforced in userspace via packet-in message, there won't be OpenFlow rules created for such rules. Therefore, assigning priority is not required."

Reply:
done